docs/sphinx/source/modelchain.rst

```diff
@@ -213,17 +220,29 @@ users' code as simple as possible.
 
 The key parts of ModelChain are:
 
-    1. The :py:meth:`ModelChain.run_model() <.ModelChain.run_model>` method
+    1. The ModelChain.run_model methods.
     2. A set of methods that wrap and call the PVSystem methods.
-    3. A set of methods that inspect user-supplied objects to determine
-       the appropriate default models.
-
-run_model
-~~~~~~~~~
-
-Most users will only interact with the
-:py:meth:`~pvlib.modelchain.ModelChain.run_model` method. The
-:py:meth:`~pvlib.modelchain.ModelChain.run_model` method, shown below,
+    3. A set of methods that can inspect user-supplied objects to infer
+       the appropriate model when a models isn't specified by the user.
+
+run_model methods
+~~~~~~~~~~~~~~~~~
+
+ModelChain provides three methods for executing the chain of models. The
+methods allow for simulating the output of the PVSystem with different
+input irradiance data:
+
+* :py:meth:`~pvlib.modelchain.ModelChain.run_model`, use when ``weather``
+  contains global horizontal, direct and diffuse horizontal irradiance ('ghi', 'dni' and 'dhi')
```

Comment:
I'd put ghi, dni, dhi in code blocks with double back ticks because they're required DataFrame column names. Same for poa* and effective_irradiance below.